tubular/dates.py

```diff
@@ -663,3 +663,344 @@ def transform(self, X):
         X[self.new_column_name] = lower_comparison & upper_comparison
 
         return X
+
+
+class DatetimeInfoExtractor(BaseTransformer):
+    """Transformer to extract various features from datetime var
+
+    Parameters
+    ----------
+    columns : str or list
+        datetime columns
+
+    include : list of str, default = ["timeofday", "timeofmonth", "timeofyear", "dayofweek"]
+        Which datetime categorical information to extract
+
+    datetime_mappings : dict, default = {}
+        Optional argument to define custom mappings for datetime values.
+        Keys of the dictionary must be contained in `include`
+        All possible values of each feature must be included in the mappings,
+        ie, a mapping for `dayofweek` must include all values 0-6;
+        datetime_mappings = {"dayofweek": {"week": [0, 1, 2, 3, 4],
```

Comment:
Currently the only place the user can find the default mappings is in the code where they are assigned - for ease of use would be better to have them either here or the example notebook (or both)